packages/editor/src/core/components/Cell/PluginComponent/index.tsx

```diff
@@ -73,7 +76,16 @@ const PluginComponent: React.FC<{ nodeId: string; hasChildren: boolean }> = ({
           }}
         >
           {Renderer ? (
-            <Renderer {...componentProps}>{children}</Renderer>
+            <Renderer {...componentProps}>
+              {hasChildren ? (
+                <div style={{ marginBottom: `${-cellSpacing}px` }}>
+                  {children}
+                </div>
```

Comment:
it might be better to just do that in the parent component. This way, you don't have to pass the insertAllowed prop.

so this new wrapper would be defined in the parent component and this component here can be left untouched

Reply:
Will do.